reports/PREOPS-prompt-error-msgs.ipynb

```diff
@@ -179,20 +182,23 @@
     "    \"Exception MeasureApCorrError: Unable to measure aperture correction\",\n",
     "    \"MatcherFailure: No matches found\",\n",
     "    \"MatcherFailure: Not enough catalog objects\",\n",
+    "    \"MatcherFailure: No matches to use for photocal\",\n",
     "    \"NoPsfStarsToStarsMatchError\",\n",
     "    \"PsfexTooFewGoodStarsError\",\n",
     "    \"RuntimeError: Cannot compute PSF matching kernel: too few sources selected\",\n",
     "    \"RuntimeError: No good PSF candidates to pass to PSFEx\",\n",
+    "    \"PsfexNoGoodStarsError\",\n",
     "    \"RuntimeError: No objects passed our cuts for consideration as psf stars\",\n",
     "    \"SIGTERM\",\n",
     "    \"ValueError: cannot convert float NaN to integer\",\n",
+    "    \"failed. Exception NotImplementedError:\",\n",
```

Comment:
It's okay to leave this here for now. But we will need to be careful when interpreting this as the underlying error still needs to be looked at.